test/util/factories/trip_factory.dart

```diff
@@ -16,5 +16,6 @@ abstract class TripFactory<T extends Trip> extends ModelFactory<T> {
     DateTime? endTime,
     int? seats,
     bool createDependencies = true,
+    bool hideInListView = false,
```

Comment:
can you move this to the spot where it is in the class definition? Definitely above `createDependencies`, that should always be the last argument for clarity